e2e-tests/testdata/echo_client/client.js

```diff
@@ -0,0 +1,912 @@
+// Code generated by the Encore devel client generator. DO NOT EDIT.
+
+
+/**
+ * Local is the base URL for calling the Encore application's API.
+ */
+export const Local = "http://localhost:4000"
+
+/**
+ * Environment returns a BaseURL for calling the cloud environment with the given name.
+ */
+export function Environment(name) {
+    return `https://${name}-slug.encr.app`
+}
+
+/**
+ * Client is an API client for the slug Encore application. 
+ */
+export default class Client {
+    cache
+    di
+    echo
+    endtoend
+    flakey_di
+    middleware
+    test
+    validation
```

Comment:
Is this normal JSON? I've never seen this pattern before. I guess it's not necessary and we should drop it?